smithed_libraries/packs/actionbar/data/smithed.actionbar/functions/impl/vanilla/container/send_message.mcfunction

```diff
@@ -0,0 +1,8 @@
+# send message for locked container
+# @s = player who clicked a lockable block
+# located at the center of the clicked block
+# run from vanilla/container/check_lock
```

Comment:
Convert comment into api doc format. There should be an example for event based docs for the `death_message` one.

Or, perhaps we add event docs directly to our docs repo. I forget what we did prior.